lib/litmus_paper/metric/haproxy_backends_health.rb

```diff
@@ -20,6 +20,7 @@ def current_health
           .inject(0) { |sum, server| sum + server["weight"].to_f }
 
         total_weight = servers
+          .select { |server| !server["status"].include?("MAINT") }
```

Comment:
```suggestion
          .reject { |server| server["status"].include?("MAINT") }
```

Just a minor nit, reads a little cleaner that way to me.